main.go

```diff
@@ -322,6 +325,15 @@ func main() {
 		return
 	}
 
+	if options.Pprof != "" {
```

Comment:
Minor/future proofing: you might want to put this code in a separate package (e.g. `debug.StartPprof(options.Pprof)`): my understanding is that once `"net/http/pprof"` is imported in package any further http.ListenAndServe calls will include the pprof endpoint whether you want it or not, so a later refactor that for example adds a metrics endpoint might end up "leaking" a pprof endpoint unintentionally.

The `init` handler in the pprof package is pretty invasive: https://golang.org/src/net/http/pprof/pprof.go#L72

Reply:
@ryanschneider I was looking for a way to do that more selectively. How would `debug.StartPprof` work though?

`net/http/pprof` injects into the "global" `net/http` handler, so that will happen even if you put it in another package, right?

As long as ListenAndServe is otherwise called with specific handlers (rather than the global handler/nil), it should be fine I think?